Incompatible with Golang 1.19

**Describe the Bug:**
App does not pass checks or compile when using golang 1.19

**Steps to Reproduce:**
```bash
make check
```
Result:
```
devtools/gettool/fetchfile.go:6:2: "io/ioutil" has been deprecated since Go 1.16: As of Go 1.16, the same functionality is now provided by package io or package os, and those implementations should be preferred in new code. See the specific function documentation for details.  (SA1019)
grafana/grafana.go:7:2: "io/ioutil" has been deprecated since Go 1.16: As of Go 1.16, the same functionality is now provided by package io or package os, and those implementations should be preferred in new code. See the specific function documentation for details.  (SA1019)
notification/twilio/twiml_test.go:4:2: "io/ioutil" has been deprecated since Go 1.16: As of Go 1.16, the same functionality is now provided by package io or package os, and those implementations should be preferred in new code. See the specific function documentation for details.  (SA1019)
smoketest/webhook_test.go:6:2: "io/ioutil" has been deprecated since Go 1.16: As of Go 1.16, the same functionality is now provided by package io or package os, and those implementations should be preferred in new code. See the specific function documentation for details.  (SA1019)
exit status 1
make: *** [check] Error 1
```

**Expected Behavior:**
make check should work without errors

**Observed Behavior:**
make check fails

**Screenshots/Stack Traces:**
If applicable, add screenshots and/or stack traces to help explain your problem.

**Application Version:**
goalert master branch

**Desktop:**
NA

**Smartphone/Tablet:**
NA

**Additional Context:**

--- a/devtools/gettool/fetchfile.go
+++ b/devtools/gettool/fetchfile.go
@@ -3,7 +3,6 @@ package main
 import (
 	"fmt"
 	"io"
-	"io/ioutil"
 	"net/http"
 	"os"
 )
@@ -19,7 +18,7 @@ func fetchFile(url string) (*os.File, int64, error) {
 		return nil, 0, fmt.Errorf("non-200 response: %s", resp.Status)
 	}
 
-	fd, err := ioutil.TempFile("", "*.zip")
+	fd, err := os.CreateTemp("", "*.zip")
 	if err != nil {
 		return nil, 0, fmt.Errorf("create temp file: %w", err)
 	}

--- a/grafana/grafana.go
+++ b/grafana/grafana.go
@@ -4,7 +4,7 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
-	"io/ioutil"
+	"io"
 	"net/http"
 	"strings"
 	"text/template"
@@ -182,7 +182,7 @@ func GrafanaToEventsAPI(aDB *alert.Store, intDB *integrationkey.Store) http.Hand
 		}
 		serviceID := permission.ServiceID(ctx)
 
-		data, err := ioutil.ReadAll(r.Body)
+		data, err := io.ReadAll(r.Body)
 		if errutil.HTTPError(ctx, w, err) {
 			return
 		}

--- a/notification/twilio/twiml_test.go
+++ b/notification/twilio/twiml_test.go
@@ -1,7 +1,7 @@
 package twilio
 
 import (
-	"io/ioutil"
+	"io"
 	"net/http"
 	"net/http/httptest"
 	"testing"
@@ -20,7 +20,7 @@ func TestTwiMLResponse(t *testing.T) {
 		resp := rec.Result()
 		assert.Equal(t, http.StatusOK, resp.StatusCode)
 		assert.Contains(t, resp.Header.Get("Content-Type"), "application/xml")
-		data, err := ioutil.ReadAll(resp.Body)
+		data, err := io.ReadAll(resp.Body)
 		assert.NoError(t, err)
 		assert.Equal(t, `<?xml version="1.0" encoding="UTF-8"?>
 <Response>
@@ -41,7 +41,7 @@ func TestTwiMLResponse(t *testing.T) {
 		resp := rec.Result()
 		assert.Equal(t, http.StatusOK, resp.StatusCode)
 		assert.Contains(t, resp.Header.Get("Content-Type"), "application/xml")
-		data, err := ioutil.ReadAll(resp.Body)
+		data, err := io.ReadAll(resp.Body)
 		assert.NoError(t, err)
 		assert.Equal(t, `<?xml version="1.0" encoding="UTF-8"?>
 <Response>
@@ -61,7 +61,7 @@ func TestTwiMLResponse(t *testing.T) {
 		resp := rec.Result()
 		assert.Equal(t, http.StatusOK, resp.StatusCode)
 		assert.Contains(t, resp.Header.Get("Content-Type"), "application/xml")
-		data, err := ioutil.ReadAll(resp.Body)
+		data, err := io.ReadAll(resp.Body)
 		assert.NoError(t, err)
 		assert.Equal(t, `<?xml version="1.0" encoding="UTF-8"?>
 <Response>
@@ -83,7 +83,7 @@ func TestTwiMLResponse(t *testing.T) {
 		resp := rec.Result()
 		assert.Equal(t, http.StatusOK, resp.StatusCode)
 		assert.Contains(t, resp.Header.Get("Content-Type"), "application/xml")
-		data, err := ioutil.ReadAll(resp.Body)
+		data, err := io.ReadAll(resp.Body)
 		assert.NoError(t, err)
 		assert.Equal(t, `<?xml version="1.0" encoding="UTF-8"?>
 <Response>
@@ -108,7 +108,7 @@ func TestTwiMLResponse(t *testing.T) {
 		resp := rec.Result()
 		assert.Equal(t, http.StatusOK, resp.StatusCode)
 		assert.Contains(t, resp.Header.Get("Content-Type"), "application/xml")
-		data, err := ioutil.ReadAll(resp.Body)
+		data, err := io.ReadAll(resp.Body)
 		assert.NoError(t, err)
 		assert.Equal(t, `<?xml version="1.0" encoding="UTF-8"?>
 <Response>

--- a/smoketest/webhook_test.go
+++ b/smoketest/webhook_test.go
@@ -7,9 +7,8 @@ import (
 	"net/http/httptest"
 	"testing"
 
-	"github.com/stretchr/testify/require"
-
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 	"github.com/target/goalert/smoketest/harness"
 )
 

--- a/smoketest/webhook_test.go
+++ b/smoketest/webhook_test.go
@@ -2,11 +2,12 @@ package smoketest
 
 import (
 	"encoding/json"
-	"github.com/stretchr/testify/require"
-	"io/ioutil"
+	"io"
 	"net/http"
 	"net/http/httptest"
 	"testing"
+
+	"github.com/stretchr/testify/require"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/target/goalert/smoketest/harness"
@@ -27,7 +28,7 @@ func TestWebhookAlert(t *testing.T) {
 	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		var alert WebhookTestingAlert
 
-		data, err := ioutil.ReadAll(r.Body)
+		data, err := io.ReadAll(r.Body)
 		require.Nil(t, err)
 
 		err = json.Unmarshal(data, &alert)

